css/style.css

```diff
@@ -29,67 +30,237 @@
+  border-right: 4.3em solid transparent;
+  border-bottom: 4.3em solid #fec25a;
+  border-left: 4.3em solid transparent;
+  position: absolute;
+  left: -80px;
+  rotate: 315deg;
+  background-blend-mode: normal, overlay, normal;
+}
+
+.red-square {
+  position: absolute;
+  width: 74px;
+  height: 74px;
+  left: 38px;
+  top: 14px;
+  background: #e35461;
+  rotate: 45deg;
   background-blend-mode: normal, overlay, normal;
 }
 
 .name {
   font-family: Playfair Display, Roboto, Arial, sans-serif;
-  font-weight: normal;
+  position: absolute;
+  width: 220px;
+  height: 100px;
+  left: 16px;
+  top: 104px;
   font-style: normal;
+  font-weight: 400;
   font-size: 50px;
   line-height: 120%;
   color: #1a2236;
-  text-align: left;
-  vertical-align: top;
-  padding: 20px 84px 0 6px;
 }
 
 .about-me {
   font-family: Poppins, Roboto, Arial, sans-serif;
-  font-weight: normal;
+  position: absolute;
+  width: 288px;
+  height: 126px;
+  left: 16px;
+  top: 294px;
   font-style: normal;
+  font-weight: 400;
   font-size: 14px;
-  line-height: 14px;
-  text-align: left;
-  letter-spacing: unset;
-  margin-top: 20px;
-  padding: 0 16px 20px 16px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  color: #1a2236;
 }
 
 .rectangle-button {
+  position: absolute;
   width: 288px;
   height: 50px;
-  padding-left: 16px;
-  right: 16px;
-  padding-bottom: 0;
-  background-color: #1a2236;
+  left: 16px;
+  top: 490px;
+  background: #1a2236;
 }
 
 .rectangle-button h1 {
+  color: #fff;
   font-family: Poppins, Roboto, Arial, sans-serif;
   font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+}
+
+.middle-image {
+  position: absolute;
+  background: url("../img/middle-section.png");
+  width: 320px;
+  height: 160px;
+  left: 0;
+  top: 582px;
+}
+
+.projects {
+  position: absolute;
+  top: 742px;
+  width: 181px;
+  height: 50px;
+  left: 16px;
+  font-family: Playfair Display, Roboto, Arial, sans-serif;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.computer-image {
+  position: absolute;
+  width: 320px;
+  height: 186px;
+  left: 0;
+  top: 882px;
+  background: url("../img/computer.png");
+  border: 0.333333px solid rgba(255, 255, 255, 0.1%);
+}
+
+.project-section {
+  position: absolute;
+  width: 320px;
+  height: 156px;
+  left: 0;
+  top: 1068px;
+  background: #1a2236;
+  border: 0.333333px solid rgba(255, 255, 255, 0.1%);
+  backdrop-filter: blur(1px);
+}
+
+.project-section h2 {
+  width: 264px;
+  height: 16px;
+  padding-left: 20px;
+  font-family: Poppins, Roboto, Arial, sans-serif;
   font-style: normal;
+  font-size: 20px;
+  font-weight: 300;
+  line-height: 80%;
+  color: #fff;
+}
+
+.grid-container {
+  position: absolute;
+  top: 1118px;
+  width: 312px;
+  height: 30px;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: auto;
+  padding-left: 12px;
+  grid-gap: 20px;
+  grid-column-gap: 10px;
+}
+
```

Comment:
- [ ] Please the content of the project should be aligned with flexbox although the works section itself should be aligned with grid